### PR TITLE
Link to spec definition of “collapse” in CSS “white-space” doc

### LINKED ISSUE #12612: Issue with "white-space": make clear what “white space is collapsed” means
MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/white-space

#### What information was incorrect, unhelpful, or incomplete?

The page states repeatedly that “Sequences of white space are collapsed.” To what? I suspect “collapsed into a single space”, or might it mean “collapses to an empty string”?

#### Specific section or headline?

Several occurrences across the page.

#### What did you expect to see?

A precise statement.

#### Did you test this? If so, how?

n/a

<details>
<summary>MDN Content page report details</summary>

* Folder: `en-us/web/css/white-space`
* MDN URL: https://developer.mozilla.org/en-US/docs/Web/CSS/white-space
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/web/css/white-space/index.md
* Last commit: https://github.com/mdn/content/commit/26f50bdaeb1228f458d21dbdf7dc110387b20daa
* Document last modified: 2021-08-12T23:59:09.000Z

</details>

### PR DESCRIPTION
**Pull request by sideshowbarker**
Fixes https://github.com/mdn/content/issues/12612

### CODE REVIEW

**Comment by jenstroeger** [comment]
Great, thank you addressing issue https://github.com/mdn/content/issues/12612 — this addition is what I was looking for 👍🏼

File: files/en-us/web/css/white-space/index.md [hunk 1]
```diff
@@ -137,6 +137,10 @@ The following table summarizes the behavior of the various `white-space` values:
 >
 > Where white space is said to _hang_, this can affect the size of the box when measured for intrinsic sizing.
 
+## Collapsing of white space
+
+The CSS Text specification contains a [Collapsing and Transformation](https://drafts.csswg.org/css-text/#white-space-phase-1) section that precisely defines what “white space is collapsed” means, including an example with an illustration. Basically, it means reducing sequences of multiple white-space characters down to a single space character — though in some cases it means reducing them to no character (the empty string).
```
Comment (wbamberg) [inline files/en-us/web/css/white-space/index.md:142]:
```suggestion
The CSS Text specification contains a [Collapsing and Transformation](https://drafts.csswg.org/css-text/#white-space-phase-1) section that precisely defines what “white space is collapsed” means, including an example with an illustration. Usually, it means reducing sequences of multiple white-space characters down to a single space character — though in some cases it means reducing them to no character (the empty string).
```

**Review by wbamberg — APPROVED** [review]
👍 thanks!

File: files/en-us/web/css/white-space/index.md [hunk 2]
```diff
@@ -137,6 +137,10 @@ The following table summarizes the behavior of the various `white-space` values:
 >
 > Where white space is said to _hang_, this can affect the size of the box when measured for intrinsic sizing.
 
+## Collapsing of white space
+
+The CSS Text specification contains a [Collapsing and Transformation](https://drafts.csswg.org/css-text/#white-space-phase-1) section that precisely defines what “white space is collapsed” means, including an example with an illustration. Usually, it means reducing sequences of multiple white-space characters down to a single space character — though in some cases it means reducing them to no character (the empty string).
```
Comment (jenstroeger) [inline files/en-us/web/css/white-space/index.md:142]: Did that link disappear between yesterday and today?

Comment (sideshowbarker) [reply files/en-us/web/css/white-space/index.md:142]: The https://drafts.csswg.org server breaks fairly regularly

I raised https://github.com/w3c/csswg-drafts/issues/7009

They usually have it working again pretty quickly

**Pull request merged**